apps/contracts/.pnpm-debug.log

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file should not be included, and we should add `.pnpm-debug.log` to the root `.gitignore`

Reply:
Updated in latest commit